WebServices/TodoWCF/TodoWCF/Constants.cs

```diff
@@ -1,8 +0,0 @@
-﻿namespace TodoWCF
-{
-	public static class Constants
-	{
-		// URL of WCF service
-		public static string SoapUrl = "https://developer.xamarin.com:8081/TodoService.svc";
```

Comment:
As above comment - leave the constant here, and specify the two different values for iOS/Android using Device.RuntimePlatform.